bowtie/benchmarks/draft2020_metaschema.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Yeah actually I felt if we have static files then atleast we would know that we just need to re run the benchmarks during deployment only when there is some change in the benchmarks folder

Reply:
Can you elaborate on what you're thinking there -- when/why would we run benchmarks during a deployment?
The things we'd run during a deployment are generally meant to ensure the deployment works correctly -- the only real "exception" is triggering a rebuild of the reports -- but that we do because it's 1) reasonably fast and 2) tests that the images we just rebuilt function.

Reply:
My bad, I was thinking that we would need to recreate the benchmark artifacts each time we deploy. But ig we can simply run the benchmarks say weekly and during deployment it would just pick the latest artifact from the last successful run right ?

Reply:
Right yeah that was what I was thinking!